docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -24,6 +25,189 @@
+		* New permissible value: `CBFA2T2`
+		* New permissible value: `CEP164`
+		* New permissible value: `CEP170B`
+		* New permissible value: `CTDP1`
+		* New permissible value: `DOT1L`
+		* New permissible value: `FRYL`
+		* New permissible value: `GLIS2`
+		* New permissible value: `HMGB3`
+		* New permissible value: `HNRNPH1`
+		* New permissible value: `INO80D`
+		* New permissible value: `JARID2`
+		* New permissible value: `PHF23`
+		* New permissible value: `PIM3`
+		* New permissible value: `PTP4A1`
+		* New permissible value: `RPS15`
+		* New permissible value: `SETD1A`
+		* New permissible value: `SLC66A2 (aka PQLC1)`
+		* New permissible value: `TOP2B`
+		* New permissible value: `UBB`
+		* New permissible value: `ZEB2`
+	* Changes made to `test_value_range`
+		* New permissible value: `<10%`
+		* New permissible value: `10–19%`
+		* New permissible value: `20–29%`
+		* New permissible value: `30–39%`
+		* New permissible value: `40–49%`
+		* New permissible value: `50–59%`
+		* New permissible value: `60–69%`
+		* New permissible value: `70–79%`
+		* New permissible value: `80–89%`
+		* New permissible value: `90–99%`
+		* New permissible value: `<1%`
+		* New permissible value: `1–49%`
+		* New permissible value: `>=50%`
+	* Changes made to `timepoint_category`
+		* New permissible value: `Preoperative`
+* Altered `pathology_detail` Entity
+	* New property: `epithelioid_cell_percent`
+	* New property: `measurement_type`
+	* New property: `measurement_unit`
+	* New property: `spindle_cell_percent`
+	* New property: `tumor_depth_descriptor`
+	* New property: `tumor_depth_measurement`
+	* New property: `tumor_infiltrating_lymphocytes`
+	* New property: `tumor_infiltrating_macrophages`
+	* New property: `tumor_length_measurement`
+	* New property: `tumor_measurement_method`
+	* New property: `tumor_shape`
+	* New property: `tumor_width_measurement`
+	* Changes made to `additional_pathology_findings`
+		* New permissible value: `Extravascular Matrix Loops`
+		* New permissible value: `Other Complex Extravascular Matrix Patterns`
+		* New permissible value: `Poorly Differentiated`
+		* New permissible value: `Well Differentiated`
+* Altered `raw_methylation_array` Entity
+	* Changes made to `platform`
+		* New permissible value: `Illumina Methylation Epic v2`
+* Altered `sample` Entity
+	* Changes made to `preservation_method`
+		* New permissible value: `EDTA`
+	* Changes made to `specimen_type`
+		* New permissible value: `Derived Cell Lines and Sorted Cells`
+* Altered `treatment` Entity
+	* New deprecated property: `treatment_anatomic_site`
+	* New property: `treatment_anatomic_sites`
+	* Changes made to `protocol_identifier`
+		* New permissible value: `P9407`
+
+
+### Bugs Fixed Since Last Release
+
+The [GDC Data Dictionary Viewer](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/) on the [GDC Documentation Site](https://docs.gdc.cancer.gov) correctly displays permissible values for array-type properties.
```

Comment:
Add an asterisk to this to make it a list.

Reply:
Asterisked 👍